File: fibonacci.java
import java.util.Scanner;
public class fibonacci 
{
	public static void main (String[] args)
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter the no. of terms : ");
		int n = sc.nextInt();
		int a=0, b=1, c;
		System.out.print(a+" "+b+" ");
		for (int i=1; i<=n; i++)
		{
			c=a+b;
			System.out.print(c+" ");
			a=b;
			b=c;
		}
	}
}

File: fibonacci.py
n = int(input("Enter the number of terms: "))  
n1 = 0  
n2 = 1  
c = 0    
if n <= 0:  
   print (0)  
elif n == 1:   
   print (n1,",", n2)  
else:      
   while c < n:
       print(n1, end = ' ')
       nth = n1 + n2     
       n1 = n2  
       n2 = nth  
       c += 1
